src/api-reference/budget/budget-category.md

```diff
@@ -3,184 +3,554 @@ title: Budget Category
+`budgetitem.read`|Read access to budget data
+
+### Request
+
+#### Headers
+
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+
+#### Parameters
+
+Name | Type | Format | Description
+-----|------|--------|------------			
+id	|	`string`	|	`path`	|	The budget category's key field.
+
+##### URI Template
 
 ```shell
-http https://us.api.concursolutions.com/budget/v4/budgetCategory/{id}  'Authorization:Bearer {YOUR ACCESS TOKEN}'
+GET  /budget/v4/budgetCategory/{id} 
+```
+
+### Response
+
+#### Status Codes
+
+* [200 OK](https://tools.ietf.org/html/rfc7231#section-6.3.1) Successful call, response is in body.
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1) The request was determined to be invalid by the server.
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3) The user does not have the necessary permissions to perform the request.
+* [404 Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4) The resource could not be found or does not exist
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1) Error message in response body
+* [504 Gateway Timeout](https://tools.ietf.org/html/rfc7231#section-6.6.5) Error message in response body
+
+#### Headers
+
+* `concur-correlationid` (Optional) is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+
+#### Payload
+
+[Budget Category](#budgetCategory)
+
+### Example
+
+#### Request
+```http
+GET https://us.api.concursolutions.com/budget/v4/budgetCategory/36047f6c-6cf6-443d-a952-39efb012acdb
+Authorization: Bearer: {YOUR ACCESS TOKEN}
+Content-Type: application/json
+```
+
+#### Response
+```http
+HTTP/1.1 200 OK
+Cache-Control: max-age=604800
+Content-Type: application/json
+Date: Wed, 06 Jul 2020 17:33:03 GMT
+Etag: "359670651"
+Expires: Wed, 13 Jul 2020 17:33:03 GMT
+Last-Modified: Fri, 09 Aug 2020 23:54:35 GMT
+Content-Length: 642
+concur-correlationid: f7b1a193-46cc-4784-9c6f-d8e1e47ecaa1
+```
+
+```json
+{
+  "name":"Marketing and Outreach",
+  "description":null,
+  "statusType":"OPEN",
+  "id":"36047f6c-6cf6-443d-a952-39efb012acdb",
+  "expenseTypes":[
+    {
+      "featureTypeCode":"PURCHASE_REQUEST",
+      "expenseTypeCode":"MKTG",
+      "id":"f35827a8-7981-4a5b-bfc3-da7ebb4665ff",
+      "name":null
+    },
+    {
+      "featureTypeCode":"EXPENSE",
+      "expenseTypeCode":"SEMNR",
+      "id":"30f16783-e50e-4ab4-b6fb-f66cc75956f2",
+      "name":null
+    },
+    {
+      "featureTypeCode":"PURCHASE_REQUEST",
+      "expenseTypeCode":"ADVT",
+      "id":"64a04928-37b0-49c8-99e8-c346e6d47825",
+      "name":null
+    }
+  ]
+} 
+```
+
+## <a name="post"></a>POST a Budget Category
+
+Save a new budget category or update an existing budget category.  
```

Comment:
CRLF between paragraph text and bulleted lists.

Reply:
👍